src/doc/book/crates-and-modules.md

```diff
@@ -105,6 +105,16 @@ parent module, the names don’t conflict: `english::greetings` and
 `japanese::greetings` are distinct, even though their names are both
 `greetings`.
 
+Before we build it, we need a bit configuration in the `Cargo.toml` file in the root folder of your project:
```

Comment:
Yes, this should make it clear that Cargo will generate this file for you, since running on the command-line is the standard way that we do this. (sounds like the plugin needs updated)